### PR TITLE
add keyring to artifactory build

### PR DESCRIPTION
**Pull request by MarkAckert**
Signed-off-by: MarkAckert <mark.ackert@broadcom.com>

Remove keyring binary from repository, pull from artifactory build

### CODE REVIEW

**Comment by vit-tomica** [comment]
I downloaded the pax file `https://zowe.jfrog.io/zowe/webapp/#/artifacts/browse/tree/General/libs-snapshot-local/org/zowe/1.13.0-PR-1453/zowe-1.13.0-pr-1453-3-20200623152544.pax`
I unpaxed it and try to run the keyring-util with:
```
USER01:/a/user01/zowe/build: >zowe-1.13.0/bin/utils/keyring-util/keyring-util
bash: zowe-1.13.0/bin/utils/keyring-util/keyring-util: cannot execute binary file: EDC5130I Exec format error.
```
The binary file probably gets corrupted somewhere on the way.
I'll try to download it from the artifactory directly, transfer to z/OS and check whether it works.

File: manifest.json.template [hunk 1]
```diff
@@ -79,11 +79,28 @@
        "explode": "true",
        "excludePatterns": ["*PR*.zip"]
     },
+    "org.zowe.keyring-utilities": {
+      "version": "~1.0.2-SNAPSHOT",
```
[the remark this answers is not in the document]
Comment (MarkAckert) [reply manifest.json.template:83]: we can swap this back to 1.0.1 instead - I was just making sure SNAPSHOT format worked OK.

**Comment by vit-tomica** [comment]
I downloaded the binary from https://zowe.jfrog.io/zowe/libs-snapshot-local/org/zowe/keyring-utilities/1.0.2-SNAPSHOT/ and I can run it on z/os fine. It may be that packaging does some conversion on it?

**Comment by vit-tomica** [comment]
That's probably what Jack pointed out in a different thread that everything in `bin/`  is supposed to be ascii
https://github.com/zowe/zowe-install-packaging/blob/94d7b3f5cef53433800bf6a71619835752530b0b/.pax/prepare-workspace.sh#L143

**Comment by MarkAckert** [comment]
OK - We can fix the encoding error by changing up some of the pre-packaging scripts. Do you still want keyring-util to be in the bin/keyring-util folder for the final PAX? Just making sure

**Comment by vit-tomica** [comment]
It can be wherever it makes more sense. I can eventually change the file location in scripts, that's no problem.

**Comment by MarkAckert** [comment]
@vit-tomica give the newest PAX a try when you can - the encoding should be right this time around. Thanks

**Comment by vit-tomica** [comment]
It works OK in the newest pax.

**Pull request merged**